### DISCUSSION TITLE
We need moderation tools to block specific keywords, urls, etc.

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#137745] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

There is a lack of available moderation tools in issues to prevent issues and/or comments containing either racist slurs, other kinds of attacks/insults or straight up malware downloads...
Last one is a major issue right now as there has been a raise of automated/hijacked user accounts posting malware through file links.

It adds a lot of extra work for people managing issues, which is especially tedious if only a single person is responsible for the repository and may not even be online to deal with those cases...

It would be beneficial to receive moderation tools to blacklist specific words, phrases, patterns, etc. to prevent the creation of harmfull issues and comments.

As an example, assume a bot is currently spreading malware through comments using `https://malware.example.com/...` as the URL to download from.
Right now would the people responsible for issue have to deal with every issue comment individually as they appear, but with moderation tools, they could add `malware.example.com/*` as a new blacklisted entry and GitHub would no longer allow issue (comments) to be created containing this text.

The option should preferably allow glob and/or regex patterns for higher customization.

This would already be a huge help in combatting spread of racism, homophobia, sexism, malware, etc. and would make the life of repository members a lot easier.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by maxisam** [comment]
TBH, I just wanna block some stupid comments like +1, me too etc... And I think this should be a default filter.
